src/improvements/template/OPCMUpgradeV200.sol

```diff
@@ -143,9 +145,18 @@ contract OPCMUpgradeV200 is OPCMBaseTask {
             // DF-30: Dispute factory owner must be l1PAOMultisig - It is checking for the OP Sepolia PAO.
             string memory expectedErrors_84532 =
                 "PROXYA-10,DF-30,PDDG-50,PDDG-DWETH-30,PDDG-DWETH-40,PDDG-ANCHORP-40,PDDG-120,PLDG-50,PLDG-DWETH-30,PLDG-DWETH-40,PLDG-ANCHORP-40";
+            
+            // Mainnet errors
+            string memory expectedErrors_10 =
```

Comment:
We will need an explanation of each error to 1) verify it's ok, 2) devs coming later so they have an explanation for why certain errors happen.

Reply:
Did that here with the new validator contract:
https://github.com/ethereum-optimism/superchain-ops/commit/6a8cd6775cee6e453c04477cd106d21ef0951bb1